client/components/board/boardFactory.js

```diff
@@ -3,27 +3,48 @@ angular.module('OpenEXP')
         const OpenBCIBoard = require('openbci-sdk');
         const ourBoard = new OpenBCIBoard.OpenBCIBoard();
 
-        var findPorts = () => {
+        // wrote a method in the openbci-sdk to return an array of ports and
+        // wrapping it in a promise here
+        var listDevices = () => {
```

Comment:
sorry, just saw that this is in the boardFactory, is there an equivalent listPort in the `connect.html`?